### ISSUE TITLE
[Accessibility]: ARIA ID attribute value must be unique for multiple Videos on Page (WCAG 4.1.2 Name, Role, Value Violation)

### ISSUE THREAD

**Issue by and-who** [kaltura/playkit-js-ui#1183]
**Description:**

If you have multiple Videos on a Page you get an Violation

> WCAG 4.1.2 Name, Role, Value
> ARIA ID attribute value must be unique

e.g. for:
`<span id="volumeButtonDescription">Click to volume control</span>`

**Reproduce:**
- Go to a Page with multiple Kaltura Videos
- Search for duplicate IDs

**Excpected:**
- No duplicate IDs, e.g. id includes instance id
